container/docker/factory.go

```diff
@@ -236,6 +239,7 @@ func Register(factory info.MachineInfoFactory, fsInfo fs.FsInfo) error {
 		machineInfoFactory: factory,
 		storageDriver:      storageDriver(sd),
 		storageDir:         storageDir,
+		ignoreMetrics:      ignoreMetrics,
```

Comment:
Nevermind, I realize that this isn't an inclusive list. In that case, I have a slight preference for `disabledMetrics` but leave it up to you.